Use mono to run autopanog.exe.

--- hugin-0.5-beta4/src/include/hugin/config_defaults.h.orig	2005-03-07 21:09:29.000000000 +0100
+++ hugin-0.5-beta4/src/include/hugin/config_defaults.h	2005-04-03 19:10:22.256760344 +0200
@@ -95,7 +95,7 @@
 #define HUGIN_APKOLOR_EXE                     "autopano.exe"
 #define HUGIN_APKOLOR_ARGS   "/allinone /path:%d /keys:%p /project:oto /name:%o /size:1024 /f %i"
 
-#define HUGIN_APSIFT_EXE                      "autopano-win32.exe"
+#define HUGIN_APSIFT_EXE                      "/usr/bin/autopanog.exe"
 #define HUGIN_APSIFT_ARGS                     "--output %o --imagelist %namefile"
 
 #define HUGIN_STITCHER_RUN_EDITOR             0l
