diff -NurbBp a/src/hugin_base/vigra_ext/ImageTransforms.h b/src/hugin_base/vigra_ext/ImageTransforms.h
--- a/src/hugin_base/vigra_ext/ImageTransforms.h	2009-09-21 03:04:06.000000000 +0200
+++ b/src/hugin_base/vigra_ext/ImageTransforms.h	2009-09-26 16:33:10.000000000 +0200
@@ -42,9 +42,7 @@
 #include "MultiThreadOperations.h"
 #include <boost/thread/thread.hpp>
 #include <boost/bind.hpp>
-#ifdef __APPLE__
-    #include <boost/function.hpp>
-#endif
+#include <boost/function.hpp>
 
 
 namespace vigra_ext
